### DISCUSSION TITLE
Margin on just the bottom of the bar

### DISCUSSION THREAD

**Discussion by twiclo** [polybar/polybar#3164] [Q&A]
I want a 1-2 pixel below the bar. The only way I've found to do this is set the border-size to 1pt but that puts padding around the entire bar. Any ideas?

**Comment by patrick96** [comment]
There are a bunch of settings that can achieve this. For one you can control borders on the four sides individually using `border-top-size`, etc.

There's also `margin-bottom` in the [`[global/wm]` section](https://github.com/polybar/polybar/wiki/Configuration#global-wm-settings), though that requires support from your window manager
